src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
+html {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 2px;
+  color: var(--text-secondary-color);
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+  padding: 32px 16px 16px;
+
```

Comment:
Ensure that this SCSS file is being compiled to CSS, as browsers cannot directly interpret SCSS files. This is important for the styles to be applied correctly.